### PR TITLE
[OCPPLAN-9067]: Adds release notes for 4.9.33

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.9
 

Issue: https://issues.redhat.com/browse/OCPPLAN-9067
 

Link to docs preview: https://deploy-preview-45798--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-33

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[[ocp-4-9-33]]
+=== RHSA-2022:2205 - {product-title} 4.9.33 bug fix and security update
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2833]: This is always the images advisory, so it should be 

```suggestion
=== RHBA-2022:2206 - {product-title} 4.9.33 bug fix and security update
```

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
...
+
+Issued: 2022-05-17
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2835]: Date should be 2022-05-18

File: release_notes/ocp-4-9-release-notes.adoc [hunk 3]
```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
...
+
+link:https://access.redhat.com/solutions/6958609[{product-title} 4.9.33 container image list]
+
+[id="ocp-4-9-33-bug-fixes"]
+==== Bug fixes
+* Previously, {product-title} 4.8 added an API for customizing the console and OAuth platform routes. This API included the `status` field to report the host names of customizable routes and the `spec` field to report the user host names in the cluster ingress configuration.  The API did not include the host names with top-level domains (TLDs) that contained decimal digits. Consequently, the users failed to specify the custom host names in the cluster ingress configuration and install the clusters with TLDs containing the decimal digits. With this update, the users can include the TLDs containing the decimal digits. As a result, {product-title} 4.8 allows users to use the cluster domains with TLDs containing the decimal digits. Additionally, {product-title} 4.8 allows the users to customize the console and OAuth routes by using the host names that are valid for routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075551[*BZ#2075551*])
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2845]:
```suggestion
* Previously, {product-title} 4.8 added an API for customizing the console and OAuth platform routes. This API included the `status` field to report the host names of customizable routes and the `spec` field to report the user host names in the cluster ingress configuration. The API did not include the host names with top-level domains (TLDs) that contained decimal digits. Consequently, the users failed to specify the custom host names in the cluster ingress configuration and install the clusters with TLDs containing the decimal digits. With this update, the users can include the TLDs containing the decimal digits. As a result, {product-title} 4.8 allows users to use the cluster domains with TLDs containing the decimal digits. Additionally, {product-title} 4.8 allows the users to customize the console and OAuth routes by using the host names that are valid for routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075551[*BZ#2075551*])
```

File: release_notes/ocp-4-9-release-notes.adoc [hunk 4]
```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
...
+
+* Previously, to configure DNS on the Azure using the installer-provisioned infrastructure (IPI) the ingress operator required an Azure
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2847]: SSG states to use "installation program" as opposed to "the installer"

File: release_notes/ocp-4-9-release-notes.adoc [hunk 5]
```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
...
+Resource Manager (ARM) endpoint. This endpoint needed to be provided through Azure Stack Hub configuration. Consequently, the ingress operator failed to configure a wildcard DNS record for ingress on Azure Stack Hub. With this update, the ingress operator can check the cluster infrastructure config object for the ARM endpoint provided by the installer. As a result, the ingress operator uses the ARM endpoint to configure the wildcard DNS record on the Azure Stack Hub. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032677[*BZ#2032677*])
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2848]: Is cluster infrastructure config object an official name? Normally, map always follows config unless it is an official name.

File: release_notes/ocp-4-9-release-notes.adoc (hunk 3 above)
Comment (xenolinux) [reply release_notes/ocp-4-9-release-notes.adoc]: Ack'ed

File: release_notes/ocp-4-9-release-notes.adoc (hunk 2 above)
Comment (xenolinux) [reply release_notes/ocp-4-9-release-notes.adoc]: Ack'ed

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (xenolinux) [reply release_notes/ocp-4-9-release-notes.adoc]: Ack'ed

File: release_notes/ocp-4-9-release-notes.adoc (hunk 5 above)
Comment (xenolinux) [reply release_notes/ocp-4-9-release-notes.adoc]: I don't think it's an official name. I changed the `config` to `configuration`.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 6]
```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[[ocp-4-9-33]]
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2832]: Should be [id="ocp-4-9-33]

File: release_notes/ocp-4-9-release-notes.adoc [hunk 7]
```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
...
+
+link:https://access.redhat.com/solutions/6958609[{product-title} 4.9.33 container image list]
+
+[id="ocp-4-9-33-bug-fixes"]
+==== Bug fixes
+* Previously, {product-title} 4.8 added an API for customizing the console and OAuth platform routes. This API included the `status` field to report the host names of customizable routes and the `spec` field to report the user host names in the cluster ingress configuration. The API did not include the host names with top-level domains (TLDs) that contained decimal digits. Consequently, the users failed to specify the custom host names in the cluster ingress configuration and install the clusters with TLDs containing the decimal digits. With this update, the users can include the TLDs containing the decimal digits. As a result, {product-title} 4.8 allows users to use the cluster domains with TLDs containing the decimal digits. Additionally, {product-title} 4.8 allows the users to customize the console and OAuth routes by using the host names that are valid for routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075551[*BZ#2075551*])
```
Comment (opayne1) [inline release_notes/ocp-4-9-release-notes.adoc:2845]: I pulled your PR down so I could consolidate this doc text a bit more.

**Comment by opayne1** [comment]
We have merged #45816. @xenolinux can you close this PR?

**Comment by xenolinux** [comment]
> We have merged #45816. @xenolinux can you close this PR?

Yes. will do

**Pull request closed without merging**